### PR TITLE
doc: Linkify issue numbers in changelog to GitHub

### PR DESCRIPTION
**Pull request by not-my-profile**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -139,10 +140,33 @@ def parse_event(env, sig, signode):
...
+        # this path trickery is needed because this script can
+        # be invoked with different working directories:
+        # * running make in docs/
+        # * running python setup.py build_sphinx in the repo root dir
+
+        with open(changelog_path) as f:
+            changelog = f.read()
+
+        def linkify(match):
+            url = 'https://github.com/sphinx-dox/sphinx/issues/' + match[1]
```
Comment (tk0miya) [inline doc/conf.py]: Found a typo.
```suggestion
            url = 'https://github.com/sphinx-doc/sphinx/issues/' + match[1]
```

**Review by tk0miya — CHANGES_REQUESTED** [review]
LGTM with nits.

File: doc/conf.py (hunk 1 above)
Comment (not-my-profile) [reply doc/conf.py]: Good catch, fixed!

**Review by tk0miya — APPROVED** [review]

**Comment by tk0miya** [comment]
Thank you for the quick update. Merging.

**Pull request merged**